From 580b2a0481bbb6238a23740e0c790596eabc7ac4 Mon Sep 17 00:00:00 2001
From: Cornelia <cornelia.bursucau@yahoo.com>
Date: Fri, 26 May 2017 15:47:20 +0300
Subject: [PATCH] paaatch

---
 CEAE/Controllers/QuestionnaireController.cs | 28 +++++++++++++++++++++++++++-
 1 file changed, 27 insertions(+), 1 deletion(-)

diff --git a/CEAE/Controllers/QuestionnaireController.cs b/CEAE/Controllers/QuestionnaireController.cs
index cc95c52..865aeed 100644
--- a/CEAE/Controllers/QuestionnaireController.cs
+++ b/CEAE/Controllers/QuestionnaireController.cs
@@ -46,7 +46,33 @@ namespace CEAE.Controllers
                         }
                     }
 
-                  
+
+                    Contact contact = new Contact();
+                    User user = new User();
+
+                    TestResult test = new TestResult();
+                    test.Date = DateTime.Now;
+
+                    Session[Constants.Session.RegisteredID] = contact.ContactID;
+                    Session[Constants.Session.UserId] = user.UserID;
+
+                    if (Session[Constants.Session.RegisteredID] != null)
+                    {
+                        test.ContactID = contact.ContactID;
+
+                    }
+
+                    else if (Session[Constants.Session.UserId] != null)
+                    {
+                        test.UserID = user.UserID;
+
+                    }
+
+                    //else nu are nici account nici nu si-a lasat email?
+
+                    //_db.TestResults.Add(test);
+                    //_db.SaveChanges();
+                    
                     jsonResponseText = "{\"status\":1,\"raspusuriCorecte\":\"" + raspunsuriCorecte + "\",\"message\":\"Model is not valid\"}";
 
 
-- 
2.12.2.windows.2

